ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/edge-zone/byo-subnet/cco-manual-security-token-service/byo-kms/provision/cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-cco-manual-security-token-service-byo-kms-provision-chain.yaml

```diff
@@ -0,0 +1,57 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-cco-manual-security-token-service-byo-kms-provision
+  steps:
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: aws-provision-kms-key
+  - ref: aws-provision-vpc-shared
+  - chain: aws-provision-edge-zone-subnet
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-aws-custom-vpc
+  - ref: ipi-conf-aws
+  - ref: ipi-conf-aws-edge-zone
+  - ref: ipi-conf-aws-kms-key
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-conf-aws-usage-info
+  - ref: ipi-conf-manual-creds
+  - chain: aws-provision-iam-user-minimal-permission
+  - ref: ipi-conf-aws-oidc-creds-provision
+  - ref: ipi-conf-manual-creds-remove-unnecessary-creds
+  - ref: ipi-install-install-aws
```

Comment:
btw, I guess we should use `ipi-install-install` in such install chain, right ?

Reply:
Because of `patch_public_ip_for_edge_node` ?

Reply:
Yes